lib/Service/AppsService.php

```diff
@@ -24,24 +24,51 @@ public function __construct(
 	public function findSupported(): array {
 		$apis = [];
 
+		$always = $this->appManager->getAlwaysEnabledApps();
 		foreach ($this->appManager->getInstalledApps() as $app) {
+			if (!$this->appManager->isEnabledForUser($app)) {
+				continue;
+			}
+			$appApis = [];
 			try {
 				$baseDir = $this->appManager->getAppPath($app);
 				$iterator = new \DirectoryIterator($baseDir);
 				foreach ($iterator as $file) {
 					if ($file->getFilename() == 'openapi.json') {
-						$apis[] = $app;
+						$appApis[] = $app;
 					} else if (str_starts_with($file->getFilename(), 'openapi-') && str_ends_with($file->getFilename(), '.json')) {
-						$apis[] = $app . '-' . substr($file->getFilename(), 8, -5);
+						$appApis[] = $app . '-' . substr($file->getFilename(), 8, -5);
 					}
 				}
 			} catch (AppPathNotFoundException) {
 			}
+			if (!empty($appApis)) {
+				$appInfo = $this->appManager->getAppInfo($app);
+				$apiInfo =[
```

Comment:
```suggestion
				$apiInfo = [
```
I think cs-fixer should complain about this as well.

Reply:
The awk eye ;)